test/compilable/test19491.d

```diff
@@ -0,0 +1,9 @@
+// https://issues.dlang.org/show_bug.cgi?id=19491
+
+class Foo {
```

Comment:
Curly braces on their own lines.

Reply:
We don't really enforce/require the style guide for test files as they get seldomly looked at. So imho it's not worth bothering about this here.

Reply:
In my opinion one should take as good care of the tests as the rest of the code.

Reply:
Fixed.